website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -484,6 +512,7 @@ Properties without a default value are required.
 | `data.attributes.vcs-repo.branch`             | string         | (previous value) | The repository branch that Terraform will execute from.                                                                                                                                                                                                                                                                                                                                     |
 | `data.attributes.vcs-repo.ingress-submodules` | boolean        | (previous value) | Whether submodules should be fetched when cloning the VCS repository.                                                                                                                                                                                                                                                                                                                       |
 | `data.attributes.vcs-repo.identifier`         | string         | (previous value) | A reference to your VCS repository in the format :org/:repo where :org and :repo refer to the organization and repository in your VCS provider. The format for Azure DevOps is `:org/:project/_git/:repo`.                                                                                                                                                                                  |
+| `data.attributes.vcs-repo.tags-regex`         | string         | (previous value) | A regular expression used to match Git tags. When this value is present and a VCS event is received that contains a matching Git tag for the regular expression, a Terraform Cloud run will be triggered.                                                                                                                                                                                   |
```

Comment:
Same rewording suggestions as the table earlier.

For the Description of `data.attributes.vcs-repo.tags-regex`, I'd suggest rewording to "Terraform Cloud triggers a run when this value is present and a VCS event occurs that contains a matching Git tag for the regular expression."

For the Description of `data.attributes.working-directory`, I'd suggest rewording to "A relative path from which Terraform executes. The path defaults to your repository's root and is typically set to a subdirectory matching the environment when multiple environments exist within the same repository."